### DISCUSSION TITLE
Can we make `@pandacss/no-hardcoded-color` a bit more forgiving?

### DISCUSSION THREAD

**Discussion by rudeayelo** [chakra-ui/eslint-plugin-panda#113] [Ideas]
As a Panda user we're in love with the ESlint plugin, but we think the [`@pandacss/no-hardcoded-color`](https://github.com/chakra-ui/eslint-plugin-panda/blob/main/docs/rules/no-hardcoded-color.md) is too restrictive and we're missing an option to globally allow native CSS keywords like:

- `transparent`
- `currentColor`
- `inherit`
- `initial`
- `unset`

Is there a chance we can have something like it or should we just stick to disabling the rule each time we use those keywords?

**Comment by anubra266** [comment]
@rudeayelo Maybe we could use adding strings to allow in the config
